rundocker.sh

```diff
@@ -0,0 +1,6 @@
+xhost local:root && \
+docker run -it --env="DISPLAY" \
+  --volume=${HOME}:/mnt/ \
+  --volume="/tmp/.X11-unix:/tmp/.X11-unix:rw" \
+  --gpus all -e NVIDIA_DRIVER_CAPABILITIES=all --privileged --net=host \
+  umrover1/travis:latest
```

Comment:
ugh i realized post merging, should this be travis:20.04?

Reply:
probaby, all though I changed this script to use $1 instead (so you can do `./rundocker <image name>`)